app/assets/javascripts/base.js.erb

```diff
@@ -123,6 +123,7 @@ var instantClick
     prog.classList.remove("showing");
 
     if (newUrl) {
+      document.querySelector('.skip-content-link').focus()
```

Comment:
```suggestion
      const skipLink = document.querySelector('.skip-content-link');
      if (skipLink) {
        skipLink.focus();
      }
```

refactor (blocking) - sorry I didn't pick this up before, but let's make this conditional _just in case_ 😅 . There _should_ always be a skip link on every page, but it feels like a good idea to be extra sure before we focus it.